### ISSUE TITLE
Change default problem timeout to 10 (seconds)

### ISSUE THREAD

**Issue by lane55** [pc2ccs/pc2v9#249]
**Feature Description**: 

Change the default problem time limit to 10 seconds

**Do you have any specific suggestions for how your feature would be ***implemented*** in PC^2?** 

One line fix in edu.csus.ecs.pc2.core.model.Problem
```
  public static final int DEFAULT_TIMEOUT_SECONDS = 30;
```
change to 
```
public static final int DEFAULT_TIMEOUT_SECONDS = 10;
```

**Additional context**:

**Comment by lane55** [comment]
When #295 is merged into develop, this can be closed.   This was tested and confirmed fixed with
commit:

commit a8193120d281ee64123516532922345b6d24f39a
Author: Douglas A. Lane <laned@ecs.csus.edu> 2022-03-19 20:25:40
Committer: Douglas A. Lane <laned@ecs.csus.edu> 2022-03-19 20:25:40
Parent: 48e6b9871dc4ed817f7baeaa71bdee8d6f51a010 (i118: moved all Viva settings into new VivaInputValidatorSettings class.)
Branches: i_295_add_variables_to_Problem_and_new_Constants, i_295_develop_work, i_295_new_executable_and_yaml_load, origin/i_295_add_variables_to_Problem_and_new_Constants, origin/i_295_develop_work, origin/i_295_new_executable_and_yaml_load

i295: Update Problem and Constants with new fields.

Dr. Clevenger Fixed i249 too (default runtime limit)
Added memory limit and sandbox variables in Problem

Updated unit tests for i249.

**Comment by lane55** [comment]
closing issue, unit test merged, issue fixed in #295  work.
